frontend/public/src/components/NotificationContainer_test.js

```diff
@@ -127,10 +126,8 @@ describe("NotificationContainer component", () => {
 
     await timeoutPromise
     wrapper.update()
-    assert.deepEqual(
-      wrapper.prop("userNotifications"),
-      omit(["message1"], messages)
-    )
+    // Due to changes in rendering due to enzyme, this now returns as undefined. Future test updates should
```

Comment:
think missing the rest of this sentence

Reply:
Good lord - I'll update that with the other comment. Train of thought derailed mid-comment